packages/page-accounts/src/test-support/MemoryStore.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright 2017-2020 @polkadot/app-accounts authors & contributors
```

Comment:
This file/folder probably needs to sit in the root. (It will be the same for most)

Reply:
That's right. However, not yet sure in which package to put common testing stuff. As it's used only in `page-accounts`, I'd leave it here for now.